packages/core/package.json

```diff
@@ -54,7 +54,7 @@
     "ganache-core": "2.10.2",
     "glob": "^7.1.4",
     "hdkey": "^1.1.0",
-    "mocha": "5.1.1",
+    "mocha": "8.0.1",
```

Comment:
This `mocha` (the one in `core`, the dependency, not the dev dependency) is exposed to the user... so isn't upgrading this one a breacking change?  Should this one be left as is?

Reply:
There could be some edge cases when doing customizable `mocha`-things (`mocha-parallel tests`?), but so far the core CLI functionality when running `truffle test` seems about the same.

I haven't checked the Library API we expose (`truffle.test`...), but last I checked it's undocumented and still considered experimental. 🤷‍♂️ 

Maybe we should dig in and outline what exactly the breaking changes are between v5.2.0 and v8.
FWIW, I don't think `truffle` has ever fully supported the complete features of `mocha`.